pageserver/src/tenant.rs

```diff
@@ -2297,6 +2298,8 @@ impl Tenant {
                 )
             })?;
 
+        // We can not use initialize here because it calls initialize_with_lock with load_layer_map=true,
+        // which cause double loading layers in LayerMap
```

Comment:
```suggestion
        // Initialize the timeline without loading the layer map, because we already updated the layer
        // map above, when we imported the datadir.
```